Bottom: 7d6dcb5a174620866d1d226cd0a6aec854fb6e73
Top:    463e4f0e15b3cee4555bfc60d077bee8f0deafdf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 23:32:07 +0100

Refresh of draw-c-expose-some-drawing-api

---

diff --git a/src/draw.c b/src/draw.c
index 1bbbca0..c2c5751 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -83,14 +83,10 @@ static bool doing_assy = false;
  */
 static void DrawEverything (BoxTypePtr);
 static void DrawPPV (int group, const BoxType *);
-static int DrawLayerGroup (int, const BoxType *);
 static void AddPart (void *);
 static void SetPVColor (PinTypePtr, int);
 static void DrawEMark (ElementTypePtr, Coord, Coord, bool);
-static void DrawMask (int side, BoxType *);
-static void DrawPaste (int side, BoxType *);
 static void DrawRats (BoxType *);
-static void DrawSilk (int side, const BoxType *);
 
 /*--------------------------------------------------------------------------------------
  * setup color for pin or via
diff --git a/src/draw.h b/src/draw.h
index ef32ebb..c0a2bb1 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -71,4 +71,10 @@ void EraseElementPinsAndPads (ElementTypePtr);
 void EraseElementName (ElementTypePtr);
 void EraseObject (int, void *, void *);
 
+void DrawLayerGroup (int side, const BoxType *drawn_area);
+void DrawPaste (int side, const BoxType *drawn_area);
+void DrawSilk (int side, const BoxType *drawn_area);
+void DrawMask (int side, const BoxType *drawn_area);
+void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
+
 #endif
